lib/metanorma/input/asciidoc.rb

```diff
@@ -35,9 +35,9 @@ def asciidoctor_validate(file, filename, options)
 
       def extract_metanorma_options(file)
         headerextract = file.sub(/\n\n.*$/m, "\n")
-        /\n:mn-document-class:\s+(?<type>[^\n]+)\n/ =~ headerextract
-        /\n:mn-output-extensions:\s+(?<extensions>[^\n]+)\n/ =~ headerextract
-        /\n:mn-relaton-output-file:\s+(?<relaton>[^\n]+)\n/ =~ headerextract
+        /\n:mn-document-class:\s+(?<type>\S[^\n]*)\n/ =~ headerextract
```

Comment:
@hmdne can Coradoc now parse this and provide headers?

Reply:
It can parse document attributes:

```ruby
        def document_attributes
          str(":") >> attribute_name.as(:key) >> str(":") >>
            space? >> attribute_value.as(:value) >> line_ending
        end
```